### PR TITLE
Journal smoke test

### PR DESCRIPTION
**Pull request by MichaelSmyth184**

### CODE REVIEW

File: dist-scripts/smoke-test.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-http=$(curl -s -o /dev/null -w "%{http_code}" -i -X GET  -H "Content-Type: application/json" -d '{"candidateId": "someCandidateId","faults":[{"id":"juncSpeed","faultsNo":10,"isSerious":false,"isDangerous":false}]}' "${JOURNAL_API_URL}")
+http=$(curl -s -o /dev/null -w "%{http_code}" -i -X GET  -H "Content-Type: application/json" "${JOURNAL_API_URL}")
```
Comment (zielinskikamil) [inline dist-scripts/smoke-test.sh]: is it working? when I try to get for just base URL I get 503

Comment (szymonrychu) [reply dist-scripts/smoke-test.sh]: yeah.. it's not and that's because the terraform managed environment is behind the devs managed one.

**Review by szymonrychu — APPROVED** [review]
Both the JOURNAL_API_URL and terraform code (environment) was refreshed.
@zielinskikamil I think it's safe to merge it :D

File: dist-scripts/smoke-test.sh (hunk 1 above)
Comment (MichaelSmyth184) [reply dist-scripts/smoke-test.sh]: Globals JOURNAL_API_URL has been updated with correct url.

**Review by zielinskikamil — APPROVED** [review]

**Pull request merged**